components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -187,20 +187,18 @@ internal inline fun <U> rustCallUniffi(syncOn: Any, callback: () -> U): U {
     synchronized(syncOn) {
         try {
             return callback()
-        } catch (e: ErrorWrapperException.Wrapped) {
+        } catch (e: ErrorWrapper.Wrapped) {
             // uniffi-generated functions currently return just a single error
             // type, which inside its message is the underlying error code
             // and message, which we can use to construct the actual error
             // from the hand-written FFI.
```

Comment:
@mhammond I had to do a bit of fixing up here because "flat" UniFFI error enums no longer carry an implicit "message" property by default. We could actually use the new rich structured error enums to avoid this hacky "code|message" setup now, but leaving that for future work.